packages/atomic/src/components/atomic-result/atomic-result.tsx

```diff
@@ -117,7 +119,10 @@ export class AtomicResult {
       .join('');
   }
 
+  private updateBreakpoints = once(() => updateBreakpoints(this.host));
+
   public render() {
+    this.updateBreakpoints();
```

Comment:
I noticed that `updateBreakpoints` was missing on results, meaning that there was a slight shift in density/font/etc at the wrong screen width.

Reply:
Yes it wasn't implemented for results, believe it would be too much to rerun for every results all the times.
Ideally it should perhaps be refactored?

Reply:
ah, I'll revert it then